wdio.conf.js

```diff
@@ -93,7 +93,11 @@ exports.config = {
               'dom.webnotifications.enabled': false,
               'media.getusermedia.screensharing.enabled': true,
               'media.navigator.permission.disabled': true,
-              'media.navigator.streams.fake': true
+              'media.navigator.streams.fake': true,
+              'media.gmp-manager.updateEnabled': true,
+              'media.gmp-gmpopenh264.autoupdate': true,
+              'media.eme.enabled': true,
```

Comment:
This is the only perf that exists
The rest have been removed

You can double check by going to `about:config` in firefox and pasting the perfs into the search bar
if the end icon is a `+` then it no longer exists

Reply:
changed it to just add 2

Reply:
`media.gmp-gmpopenh264.enabled` and `media.eme.enabled` are true by default, so adding these don't change anything

Reply:
This PR could probably be closed since the changes don't change default firefox behavior
Unless saucelabs changes these on their end and need to be reenabled
Or you want to go through our other firefox configs and see what needs to be taken out if the perf don't exist or if the value they're override is the same value by default